packages/cli/src/cli/github/github.ts

```diff
@@ -50,16 +55,17 @@ export class Github {
    *
    * @returns {ref} github references or the new created branch
    */
-  async createBranch(branch: string, ref: string): Promise<void> {
+  async createBranch(branch: string, ref: string): Promise<string> {
     // Get the latest sha from master branch
     const master = await this.octokit.rest.git.getRef({ owner, repo, ref: `heads/${base}` });
     if (!this.isOk(master.status)) throw new Error('Failed to get master head.');
     const sha = master.data.object.sha;
 
     // Create new branch from the latest master
     this.logger.info({ branch }, 'GitHub: Create branch');
-    const response = await this.octokit.rest.git.createRef({ owner, repo, ref, sha });
+    const response = await this.octokit.rest.git.createRef({ owner, repo, ref: `refs/${ref}`, sha });
```

Comment:
This is a bug or difference when using octokit rest api. When `createRef `we need use `refs/heads/branch`. But for getRef and updateRef, we need use `heads/branch``